db/migrate/20131126212804_create_industries.rb

```diff
@@ -2,6 +2,7 @@ class CreateIndustries < ActiveRecord::Migration
   def change
     create_table :industries do |t|
       t.string :name
+      t.string :protege_id
```

Comment:
You cannot edit an existing migration in Rails. You must add a new one. I think you should read the [documentation on migrations](http://edgeguides.rubyonrails.org/active_record_migrations.html) then come back and adjust this PR. We don't use ActiveRecord in webservices, so I can understand why you're unfamiliar with this process.